master/static/srv/proto_get_trials_plus.sql

```diff
@@ -0,0 +1,125 @@
+WITH const AS (
```

Comment:
@dzhu do you mind also 👀 this query. since I had a hand in crafting it, i'd hate for it to have an issue.